xml/mysql_banners.xml

```diff
@@ -1035,6 +1192,101 @@
     <param pos="0" name="os.version" value="10.04"/>
     <param pos="0" name="os.cpe23" value="cpe:/o:canonical:ubuntu_linux:10.04"/>
   </fingerprint>
+  <fingerprint pattern="^(?:\d{1,2}\.\d{1,3}\.[a-f\d]{1,3}-)?(\d{1,2}\.\d{1,3}\.[a-f\d]{1,4})-MariaDB-\dubuntu\d(?:-log)?$" flags="REG_ICASE">
+    <description>MariaDB running on a Ubuntu Host</description>
+    <example service.version="10.0.17">5.5.5-10.0.17-MariaDB-0ubuntu1</example>
+    <example service.version="10.3.22">5.5.5-10.3.22-MariaDB-1ubuntu1-log</example>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="service.vendor" value="MariaDB"/>
+    <param pos="0" name="service.family" value="MySQL"/>
+    <param pos="0" name="service.product" value="MariaDB"/>
+    <param pos="0" name="service.edition" value="Galera Cluster"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:mariadb:mariadb:{service.version}"/>
+    <param pos="0" name="os.vendor" value="Ubuntu"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="Linux"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:canonical:ubuntu_linux:10.04"/>
```

Comment:
How do we know this is Ubuntu `10.04`?  Also, probably not Galera. Copy-pasta?

Reply:
it was me getting to keen with cmd + v

Reply:
It happens to literally everyone so no worries.

As a note, I'd recommend change the description here to:
`MariaDB MariaDB on Ubuntu 10.04 (Lucid Lynx)`

Reply:
@jpincombe-r7 - Going back to my original comment: How do we know this is Lucid? I many have jumped the gun on my last comment.

Reply:
I'll land the PR and come back an fix this.

Reply:
Ah, I see the adjustment to the example.